runtimes/parachain/src/lib.rs

```diff
@@ -713,6 +713,7 @@ impl parachain_staking::Config for Runtime {
 	type MinDelegation = MinDelegatorStk;
 	type MinDelegatorStk = MinDelegatorStk;
 	type MaxUnstakeRequests = MaxUnstakeRequests;
+	type WeightInfo = parachain_staking::default_weights::SubstrateWeight<Runtime>;
```

Comment:
I would like to stress the point that the benchmarks are not final. The unit type conveys this message, i think. Not sure why we even have the substrate Weight in the template. Maybe we should remove that. 😅 

```suggestion
	type WeightInfo = ();
```

Reply:
Don't we want the weights to be applied to the runtime for the Audit and testing though?